fabfile.py

```diff
@@ -165,6 +165,40 @@ def listdir(path):
     return run("ls " + path).split()
 
 
+@task
+def get_snapshot_etalab_server():
+    import ovh
+    import json
+    client = ovh.Client(
+        endpoint=env.endpoint,
+        application_key=env.application_key,
+        application_secret=env.application_secret,
+        consumer_key=env.consumer_key,
+    )
+
+    print "Welcome", client.get('/me')['firstname']
+    result = client.get('/vps/' + env.vps_identifier + '/snapshot')
+    print json.dumps(result, indent=4)
+
+
+@task
+def set_snapshot_etalab_server():
```

Comment:
Maybe make the name of these 2 commands more generic? Something like create_ovh_vps_snapshot and get_ovh_vps_snapshot ? We have abstracted all our fabfile commands to not mention a perticular server we own but rather a type of server that has some characteristics (sentry, piwik, certbot, etc).